File: 백준/Bronze/8958. OX퀴즈/OX퀴즈.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder sb = new StringBuilder();
        int T = Integer.parseInt(br.readLine());
        for(int tc = 0; tc < T; tc++) {
            String S = br.readLine();
            int cnt = 0;
            int ans = 0;
            for(int i = 0; i < S.length(); i++) {
                if(S.charAt(i) == 'O') {
                    ans += ++cnt;
                } else {
                    cnt = 0;
                }
            }
            sb.append(ans + "\n");
        }
        System.out.println(sb.toString().trim());
    }
}

File: 백준/Bronze/8958. OX퀴즈/OX퀴즈.js
let [T, ...input] = require('fs').readFileSync('/dev/stdin').toString().trim().split('\n').map(S => S.trim());
T = Number(T);
const answer = [];
for(let tc = 0; tc < T; tc++) {
  let cnt = 0;
  let ans = 0;
  for(let i = 0; i < input[tc].length; i++) {
    if(input[tc][i] === 'O') ans += ++cnt;
    else cnt = 0;
  }
  answer.push(ans);
}
console.log(answer.join('\n'));